src/5e-SRD-Races.json

```diff
@@ -5,9 +5,10 @@
     "speed": 25,
     "ability_bonuses": [
       {
+        "index": "con",
         "name": "CON",
-        "url": "/api/ability-scores/con",
-        "bonus": 2
+        "bonus": 2,
```

Comment:
It's weird that the bonus is in here with this. But probably not something that needs to be done in this PR. `bonus` at this level makes sense but it feels like the rest of the data should be keyed off something?

Reply:
I agree. That's something I already mentioned in the original PR comment.

Reply:
Apparently my reading comprehension is terrible.